core/src/Instance/AadAuthority.cs

```diff
@@ -57,11 +57,13 @@ public AadAuthority(string authority, bool validateAuthority) : base(authority,
         protected void UpdateCanonicalAuthority()
         {
             UriBuilder uriBuilder = new UriBuilder(CanonicalAuthority);
+            // will not be needed after authority aliasses will be added
```

Comment:
dead code - remove or uncomment

Reply:
Petro, I believe this is the Authority Migration code. I suppose this will come in a follow up PR before completing cache work. I suggest leaving but ensuring you comment here that this is expected to complete right after this PR completes.

---
In reply to: [199546203](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199546203) [](ancestors = 199546203)